shared/components/Community/index.js

```diff
@@ -68,6 +46,7 @@ Community.propTypes = {
   mailingListSummary: PropTypes.string,
   events: PropTypes.arrayOf(PropTypes.shape(NextCommunityEvent.propTypes)),
   featuredEvent: PropTypes.shape(EventDetails.propTypes),
+  futureEvents: FutureEvents.propTypes.events,
```

Comment:
This feels a bit weird - PropTypes.arrayOf(PropTypes.shape(FutureEvent.propTypes)), is this right?